ml/src/main/scala/ai/h2o/sparkling/ml/algos/H2OGridSearch.scala

```diff
@@ -105,7 +106,10 @@ class H2OGridSearch(override val uid: String)
       internalFeatureCols: Array[String]): Array[H2OMOJOModel] = {
     val conf = H2OContext.ensure().getConf
     val endpoint = RestApiUtils.getClusterEndpoint(conf)
-    val skippedFields = Seq((classOf[GridSchemaV99], "summary_table"), (classOf[GridSchemaV99], "scoring_history"))
+    val skippedFields = Seq(
+      (classOf[GridSchemaV99], "cross_validation_metrics_summary"),
```

Comment:
Is this line fixing the bug?

Reply:
Yup -> H2O is sending invalid JSON ( the same issue you bumped into and which led us to introducing of `skippedFields` field)